commons-civil/src/main/java/uk/gov/hmcts/reform/hmc/client/HearingsApi.java

```diff
@@ -35,7 +37,17 @@ HearingGetResponse getHearingRequest(
     PartiesNotifiedResponses getPartiesNotifiedRequest(
         @RequestHeader(AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuthorization,
-        @PathVariable("id") String id
+        @RequestParam("id") String id
```

Comment:
not a problem but make sure you rebase before merging with master

it should be 
```@PathVariable("id") String id```